python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -105,7 +105,8 @@ def __copy__(self, memo=None):
             else:
                 this_dict[elem] = None
         #obj.__setstate__(this_dict)
-        obj.getParent(self._getParent())
+        if self._getParent() is not None:
+            obj._setParent(self._getParent())
```

Comment:
Is this really needed?
If the object has no parent it's copy should still have no parent, or is the _parent correctly initialized to ```None```? (I suspect it might be from looking at the ```cls()```)